CHANGE_MANAGEMENT.md

```diff
@@ -0,0 +1,148 @@
+
+## **Who can be part of DQIC?**
+Any interested stakeholder that is involved in digital quality measurements is welcome; however, participation as a committer or DQIC Steering Comittee (DSC) member requires active participation and contributions as Committers and DSC members, compliance with the DQIC charter, and a monthly membership fee. Contact DQIC’s Executive Committee (https://www.ncqa.org/digital-quality-implementers-community/) to learn more about official membership in DQIC . All products produced by DQIC will be open-source and available to the public.
+Although these processes are adapted from the [OSSWatch Meritocratic Model](http://oss-watch.ac.uk/resources/meritocraticgovernancemodel), this documentation is a formalization of existing processes involving relevant stakeholders.
+
+# Roles
+ There are four roles in the DQIC. 
+ 
+ ## Stakeholders
+Anyone can be a stakeholder; there are no special requirements; however, the most important stakeholders of this community consist of:
+- Members of government agencies that review and provide guidance as needed to support maturity of digital quality standards
+- Platform or technology owners that provide software to report on Digital Quality based on the Clinical Quality Language (CQL),
+- Payer and provider systems that use software to report on Digital Quality, and
+- Quality standard developers that maintain measures of quality in healthcare.
+
+Without these key stakeholders, the community would have limited value. Stakeholder participation is on a voluntary basis, but DQIC requests stakeholders to actively participate in the project and community as much as possible. Stakeholder contributions enable DQIC to ensure that they are satisfying the requirements of those stakeholders. Common stakeholder contributions include (but are not limited to):
+
+- Evangelism of the project (e.g. a link on a website and word-of-mouth awareness)
+- Informing developers of strengths and weaknesses from a new stakeholder perspective
+- Providing moral support (a ‘thank you’ goes a long way)
+- Stakeholders who continue to engage with the project and its community will often become more and more involved. Such stakeholders may find themselves becoming contributors, as described in the next section.
+
+## Contributors
+
+Contributors are community members who contribute in concrete ways to DQIC. Anyone can become a contributor, and contributions can take many forms. There is no expectation of commitment to the project, no specific skill requirements and no selection process.
+
+In addition to their actions as stakeholders, contributors may also find themselves doing one or more of the following:
+
+- Contributing to the [Cinical Quality Language](https://github.com/cqframework/clinical_quality_language/tree/master) project
```

Comment:
This should be [CQL Tests Repository](https://github.com/cqframework/cql-tests) (this repository), right?

Reply:
```suggestion
- Contributing to the [CQL Tests](https://github.com/cqframework/cql-tests) repository (this project)
```